planning/behavior_velocity_planner/src/scene_module/occlusion_spot/scene_occlusion_spot_in_public_road.cpp

```diff
@@ -56,6 +56,9 @@ bool OcclusionSpotInPublicModule::modifyPathVelocity(
     param_.v.max_stop_accel = planner_data_->max_stop_acceleration_threshold;
     param_.v.v_ego = planner_data_->current_velocity->twist.linear.x;
     param_.v.a_ego = planner_data_->current_accel.get();
+    param_.detection_area_max_length = planning_utils::calcJudgeLineDistWithJerkLimit(
```

Comment:
Could you add a description about detection_area_length to the document?

Reply:
updated at 
[1b3e3e3](https://github.com/autowarefoundation/autoware.universe/pull/354/commits/1b3e3e305c351bef940936f066dc53b713d67475)